src/next-app/pages/appointment-details.tsx

```diff
@@ -0,0 +1,280 @@
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={appointment.customer.phone_number}
+                  fieldName="customer.phone_number"
+                  fieldLabel="Phone Number"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={appointment.customer.email}
+                  fieldName="customer.email"
+                  fieldLabel="Email"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <h2 className="form-header">Vehicle Information</h2>
+                <CardTextField
+                  fieldValue={vehicle.manufacturer}
+                  fieldName="vehicle.manufacturer"
+                  fieldLabel="Manufacturer"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.model}
+                  fieldName="vehicle.model"
+                  fieldLabel="Model"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.year}
+                  fieldName="vehicle.year"
+                  fieldLabel="Year"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.color}
+                  fieldName="vehicle.color"
+                  fieldLabel="Color"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.vin}
+                  fieldName="vehicle.vin"
+                  fieldLabel="VIN"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                {inEdit && <Button type="submit" title="Save" width={'100%'}></Button>}
+              </div>
+            </form>
+          </FormikProvider>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const getServerSideProps: GetServerSideProps<{}> = async (context) => {
+  const { id } = context.query;
+  //const id = 1;
+  const parsedCookies = cookie.parse(String(context.req.headers.cookie));
+  const access_token = parsedCookies.access;
+  try {
+    const appointment = await axios.get(`${apiUrl}/shops/appointments/${id}`, {
```

Comment:
This request doesn't work (it gives a 404) when using shop owner JWT instead of customer JWT. Let's talk to Himanshu about this in meeting today.

Reply:
According to the meeting discussion, it looks like the problem is not with this request, but with the vehicles request.